engine/src/main/java/org/operaton/bpm/engine/ProcessEngines.java

```diff
@@ -163,8 +168,25 @@ public static void unregister(ProcessEngine processEngine) {
     processEngines.remove(processEngine.getName());
   }
 
+  /**
+   * Check if the given process engine with that name is already registered.
+   */
+  public static boolean isRegisteredProcessEngine(String processEngineName) {
+    return processEngines.containsKey(processEngineName);
+  }
+  
+  /**
+   * Returns a random name for a ProcessEngine that is not yet used. This name might be used for a new ProcessEngine without explicitly name it.
+   */
+  public static String newRandomUnusedProcessEngineName() {
```

Comment:
This is only used by tests. It should not be in production code. Move to some test utility.

Reply:
fixed